packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -191,20 +191,20 @@ export interface ContainerRuntimeMessage {
 }
 
 // Consider idle 5s of no activity. And snapshot if a minute has gone by with no snapshot.
-const IdleDetectionTime = 5000;
+const IdleDetectionTime = 15000;
```

Comment:
I'm not sure why you are changing it - it will have pretty bad outome in terms of increasing number of trailing ops (or ops per summary)

Reply:
Or is this a reflection of current setting from service?

Reply:
Let's make a change that is noop change in terms of policy changes, i.e. not mix what we want to do (lower maxOps: 100) vs, what is actual change in SPO.
If would be also confirm that Azure settings are the same as SPO before we make this noop change

Reply:
yes, I was changing to match what we currently get  from the service.  Ok will revert these.